packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -147,6 +147,46 @@ export const useChatMessageRenderStyles = makeStyles({
   }
 });
 
+const bodyWithPlaceholderImage = (myMessage: boolean = false): GriffelStyle => {
+  return {
+    '& img[src=""]': {
+      display: 'block',
+      position: 'relative',
+      marginBottom: '5px',
+      width: myMessage ? '12rem' : undefined,
+      height: myMessage ? '12rem' : undefined
```

Comment:
why is it for "myMessage" that we hard code the width and height to be 12rem?

Reply:
@jpeng-ms We're planning not to add width and height to the inline image we send out. In that case, the placeholder don't have the real image size to render with. This is not the case for remote messages we receive from Teams, it comes with width and height

Reply:
can you add some comments so we don't forget about the reasons?